src/database/models/user.js

```diff
@@ -50,6 +50,10 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.BOOLEAN,
       defaultValue: false
     },
+    verifiedToken: {
+      allowNull: true,
+      type: DataTypes.STRING
```

Comment:
why are we using a string instead of a boolean type?

Reply:
i meant to change this, i will change it now

Reply:
done